examples/quantization_re_id.py

```diff
@@ -17,6 +17,10 @@
 def argument_parser():
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument('--trained-model', type=str, default='test', help="path to trained model stored")
+    parser.add_argument('--convbn',
+                        type=str,
+                        default=True,
```

Comment:
If we set this value to True, than we should merge it at the same time with training instructions update in the FaceReID repository.